platform_versioned_docs/version-24.1.0/compute-envs/hpc.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: "HPC compute environments"
+description: "Instructions to set up HPC compute environments in Seqera Platform"
+date: "11 May 2023"
+tags: [slurm, lsf, pbs, grid, altair, ibm, moab, slurm, compute environment]
+---
+
+Seqera Platform streamlines the deployment of Nextflow pipelines into both cloud-based and on-prem HPC clusters and supports compute environment creation for the following management and scheduling solutions:
+
+- [Altair PBS Pro](https://www.altair.com/pbs-professional/)
+- [Grid Engine](https://www.altair.com/grid-engine/)
+- [IBM Spectrum LSF](https://www.ibm.com/products/hpc-workload-management/details) (Load Sharing Facility)
+- [Moab](http://docs.adaptivecomputing.com/suite/8-0/basic/help.htm#topics/moabWorkloadManager/topics/intro/productOverview.htm)
+- [Slurm](https://slurm.schedmd.com/overview.html)
+
+## Requirements
+
+To launch pipelines into an **HPC** cluster from Seqera, the following requirements must be satisfied:
+
+- The cluster should allow outbound connections to the Seqera web service.
+- The cluster queue used to run the Nextflow head job must be able to submit cluster jobs.
+- The Nextflow runtime version **21.02.0-edge** (or later) must be installed on the cluster.
+
+## Credentials
+
+Seqera requires SSH access to your HPC cluster to run pipelines. Use [managed identities](../credentials/managed_identities.mdx) to enable granular access control and preserve individual cluster user identities. If you do not need to grant multiple users access to your cluster, use workspace [SSH credentials](../credentials/ssh_credentials.mdx) for cluster login. 
```

Comment:
>If you do not need to grant multiple users access to your cluster, use workspace [SSH credentials](../credentials/ssh_credentials.mdx) for cluster login.

This is not really like this, it's more like if you're ok of having a service-account access to your hpc where all tower users use the same hpc cluster user to run jobs (so they will all share same identity, have access to the same files and are indistinguishable one from the other on the hpc side), then it's ok to just use one share SSH credential. 
Not sure how to make it more concise/precise though

Reply:
I'd play up the auditing and/or side. e.g. "Use managed identities if you need HPC-side auditing solutions to be able to:
1. Identify the actions taken by named individual
2. Restrict access to portions of the file system to specific users only (not shared groups)"